WooCommerce/Classes/ViewRelated/Orders/Order Creation/OrderForm.swift

```diff
@@ -353,6 +353,7 @@ struct OrderForm_Previews: PreviewProvider {
 private extension ProductSelectorView.Configuration {
     static let addProductToOrder: Self =
         .init(multipleSelectionsEnabled: ServiceLocator.featureFlagService.isFeatureFlagEnabled(.productMultiSelectionM1),
+              clearSelectionEnabled: !ServiceLocator.featureFlagService.isFeatureFlagEnabled(.productMultiSelectionM1),
```

Comment:
Do we need to add another flag here? Can we rely on the `multipleSelectionsEnabled` enable clearing selection?

Reply:
> Do we need to add another flag here?

We don't! I switched it to `false` by default on https://github.com/woocommerce/woocommerce-ios/pull/8995/commits/d7a18284aa0f1d3ef6759b5650b331da2ceddd67 . The initial rationale when passing the flag was to account for single-product selection, but the original view doesn't contain the "Clear Selection" button to begin with, so passing the feature flag to `clearSelectionEnabled` is not needed.